app/src/lib/stores/app-store.ts

```diff
@@ -696,6 +698,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     this.accountsStore.onDidUpdate(accounts => {
       this.accounts = accounts
+      const endpointTokens = accounts.map<EndpointToken>(
+        ({ endpoint, token }) => ({ endpoint, token })
+      )
```

Comment:
I think it would have compiled even without making it an interface but given that `accounts` contain instances of `Account` classes I wanted to explicitly convert this to dumb objects before passing the IPC serialization boundary similar to https://github.com/desktop/desktop/blob/ef31b214980582a077bb2036002bc92daec6ee9b/app/src/ui/main-process-proxy.ts#L252-L258

It just gives us a little more explicit control over what we're sending is all.